spec/spec_helper.rb

```diff
@@ -20,6 +20,20 @@
 
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |file| require file }
 
+Capybara.register_driver :chrome do |app|
+  args = %w[no-sandbox disable-gpu disable-dev-shm-usage]
+  args << (ENV['NONHEADLESS'] == 'false' ? 'headless' : 'non-headless')
```

Comment:
```suggestion
  args << (ENV['HEADLESS'] == 'true' ? 'headless' : 'non-headless')
```

maybe that's simpler? I find the double negation very confusing

Reply:
:+1: